website/docs/cloud-docs/users-teams-organizations/users.mdx

```diff
@@ -17,12 +17,28 @@ User accounts belong to individual people. Each user can be part of one or more
+
+To log in with your HCP account, go to the [Sign In to Terraform Cloud](https://app.terraform.io/) page and click **Continue with HCP account**. Terraform Cloud may ask if you want to link your account.
+
+### Linked HCP and Terraform Cloud Accounts
+
+The first time you log in with your HCP credentials, Terraform Cloud searches for existing Terraform Cloud accounts with the same email address. If there is an unlinked account, Terraform Cloud asks if you want to link it to your HCP account. If there is no existing Terraform Cloud account with that email address, Terraform Cloud creates one and automatically links it to your HCP account.
+
+After you link your HCP and Terraform Cloud accounts, you can only log in with your HCP credentials. You must unlink the accounts to resume logging in with your Terraform Cloud username and password.
+
+You cannot unlink an autogenerated Terraform Cloud account from your HCP account. You can unlink a pre-existing Terraform Cloud account on the [HCP Account Linking page](#hcp-account-linking) in your User settings.
+
 ## Creating an Account
 
-Creating an account requires a username, an email address, and a password. To use Terraform Cloud, you must create an account through one of the following methods:
+To use Terraform Cloud or Enterprise, you must create an account through one of the following methods:
 
 - **Invitation Email:** When a user sends you an invitation to join an existing Terraform Cloud organization, the email includes a sign-up link. After you create an account, you will automatically join that organization and can begin using Terraform Cloud.
-- **Sign-Up Page:** For Terraform Cloud, go to [`https://app.terraform.io/signup/account`](https://app.terraform.io/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
+- **Sign-Up Page:** Creating an account requires a username, an email address, and a password. For Terraform Cloud, go to [`https://app.terraform.io/public/signup/account`](https://app.terraform.io/public/signup/account). For Terraform Enterprise, go to `https://<TFE HOSTNAME>/public/signup/account`. After you create an account, you will not belong to any organizations. To begin using Terraform Cloud, you can either [create an organization](/cloud-docs/users-teams-organizations/organizations#creating-organizations) or ask an organization owner to send you an invitation email to join their organization.
+
+-> **Note:** For Terraform Cloud, you may use [your existing HCP account](#log-in-with-your-hashicorp-cloud-platform-account) by clicking **Continue with HCP account** button from the **Sign-Up Page**.
```

Comment:
I'm okay with this note here - thank you for flagging it for me! I think this is a case when it makes sense (I just like to make sure we don't overuse notes throughout the docs).  However, i'm recommending a couple of tiny tweaks:

- I want users who may be interested in this method to actually follow the link to the section where we talk about it/the implications, rather than just blindly doing it. So I took out the "how" part. 
- I think we should say again here that this is recommended. 
- Finally - I think we should move this to the top of the section, just under the header. I think it would be good for users to see this note _before_ they make a separate account. This helps prevent them from going through the steps to create a TFC account and not even read down to this section where we recommend logging in with HCP instead. 

```suggestion
-> **Note:** For Terraform Cloud, we recommend [logging in with your HCP account](#log-in-with-your-hashicorp-cloud-platform-account) instead of creating a separate Terraform Cloud account. 
```